lib/constants.go

```diff
@@ -581,6 +596,22 @@ type DeSoParams struct {
 	// attack the bancor curve to any meaningful measure.
 	CreatorCoinAutoSellThresholdNanos uint64
 
+	// StakeLockupEpochDuration is the number of epochs that a
+	// user must wait before unlocking their unstaked stake.
+	// TODO: Move this to GlobalParamsEntry.
+	StakeLockupEpochDuration uint64
+
+	// ValidatorJailEpochDuration is the number of epochs that a validator must
+	// wait after being jailed before submitting an UnjailValidator txn.
+	// TODO: Move this to GlobalParamsEntry.
+	ValidatorJailEpochDuration uint64
+
+	// When registering, a validator must include a VotingSignatureBlockHeight within CurrentBlockHeight
+	// to CurrentBlockHeight + ValidatorVotingSignatureBlockHeightWindow blocks. This is to prevent
+	// validator registration replay attacks.
+	// TODO: Move this to GlobalParamsEntry.
+	ValidatorVotingSignatureBlockHeightWindow uint64
```

Comment:
I don't think this is needed since the transaction will natively include a nonce with an expiration block height right? I think you can delete this. If they want to prevent replay, they can just set a smaller ExpirationBlockHeight in their nonce.